consensus/src/chained_bft/block_storage/block_tree.rs

```diff
@@ -28,6 +29,17 @@ use std::{
 };
 use types::crypto_proxies::LedgerInfoWithSignatures;
 
+struct BlockHash {
+    digest: HashValue,
+    li_sig: HashMap<HashValue, LedgerInfoWithSignatures>,
+}
+impl BlockHash {
+    /// Store BlockId to LedgerInfo hash mapping
```

Comment:
This comment is confusing: the function doesn't store any new value to a map, it simply initializes the fields. Tbh we could do it without this constructor at all :)